src/app/components/JournalSection.tsx

```
@@ -48,7 +48,10 @@ export function JournalSection() {
 
         <div className="max-w-8xl mx-auto">
           <div className="grid grid-cols-1 md:grid-cols-3 gap-8 mb-12">
-            {journalPosts.map((post, index) => (
+            {journalPosts.map((post, index) => {
+              const [isPressed, setIsPressed] = useState(false);
```

Biome (2.1.2)

[error] This hook is being called from a nested function, but all hooks must be called unconditionally from the top-level component.

For React to preserve state between calls, hooks needs to be called unconditionally and always in the same order.
See https://reactjs.org/docs/hooks-rules.html#only-call-hooks-at-the-top-level

(lint/correctness/useHookAtTopLevel)